systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeScramShaST.java

```diff
@@ -4,225 +4,115 @@
 import org.apache.logging.log4j.Logger;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import io.strimzi.systemtest.resources.KubernetesResource;
-import io.strimzi.systemtest.resources.crd.KafkaBridgeResource;
-import io.strimzi.systemtest.resources.crd.KafkaResource;
-import io.strimzi.systemtest.resources.crd.KafkaTopicResource;
-import io.strimzi.systemtest.resources.crd.KafkaUserResource;
 
-import static io.strimzi.systemtest.Constants.BRIDGE;
-import static io.strimzi.systemtest.Constants.EXTERNAL_CLIENTS_USED;
-import static io.strimzi.systemtest.Constants.NODEPORT_SUPPORTED;
-import static io.strimzi.systemtest.Constants.REGRESSION;
-import static io.strimzi.systemtest.bridge.HttpBridgeST.NAMESPACE;
+import static io.strimzi.systemtest.Constants.INTERNAL_CLIENTS_USED;
 import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
-@Tag(BRIDGE)
-@Tag(REGRESSION)
-@Tag(NODEPORT_SUPPORTED)
-@Tag(EXTERNAL_CLIENTS_USED)
-@ExtendWith(VertxExtension.class)
+@Tag(INTERNAL_CLIENTS_USED)
 class HttpBridgeScramShaST extends HttpBridgeAbstractST {
     private static final Logger LOGGER = LogManager.getLogger(HttpBridgeScramShaST.class);
+    private static final String NAMESPACE = "bridge-scram-sha-cluster-test";
 
-    private String bridgeHost = "";
-    private int bridgePort = Constants.HTTP_BRIDGE_DEFAULT_PORT;
+    private String kafkaClientsPodName;
 
     @Test
-    void testSendSimpleMessageTlsScramSha() throws Exception {
-        int messageCount = 50;
+    void testSendSimpleMessageTlsScramSha() {
         // Create topic
         KafkaTopicResource.topic(CLUSTER_NAME, TOPIC_NAME).done();
 
-        JsonObject records = BridgeUtils.generateHttpMessages(messageCount);
-        JsonObject response = BridgeUtils.sendMessagesHttpRequest(records, bridgeHost, bridgePort, TOPIC_NAME, client);
-        KafkaBridgeUtils.checkSendResponse(response, messageCount);
+        KafkaClientsResource.producerStrimziBridge(producerName, bridgeServiceName, bridgePort, TOPIC_NAME, MESSAGE_COUNT).done();
+        ClientUtils.waitForClientSuccess(producerName, NAMESPACE, MESSAGE_COUNT);
 
-        BasicExternalKafkaClient kafkaClient = new BasicExternalKafkaClient.Builder()
+        InternalKafkaClient internalKafkaClient = new InternalKafkaClient.Builder()
             .withTopicName(TOPIC_NAME)
             .withNamespaceName(NAMESPACE)
             .withClusterName(CLUSTER_NAME)
+            .withMessageCount(MESSAGE_COUNT)
             .withKafkaUsername(USER_NAME)
-            .withMessageCount(messageCount)
-            .withSecurityProtocol(SecurityProtocol.SASL_SSL)
+            .withUsingPodName(kafkaClientsPodName)
             .build();
 
-        assertThat(kafkaClient.receiveMessagesTls(), is(messageCount));
```

Comment:
The point of this test was to send the encrypted (tls) and moreover with support of simple authentication (scram-sha). You have changed to plain communication. Why? I am assuming that you wanted to just test thee scram-sha with plain communication. If this is the case please change the name of the test :)

Reply:
My bad! I will change it :) thanks